tests/eas/basic_check.py

```diff
@@ -0,0 +1,306 @@
+                            "CONFIG_CGROUPS" : "CGroups enabled",
+                            "CONFIG_CGROUP_SCHEDTUNE" : "SchedTune CGroup Controller present",
+                            "CONFIG_SMP" : "SMP enabled",
+                            "CONFIG_SCHED_MC" : "MC-level scheduler domains",
+                            "CONFIG_CPU_FREQ" : "CPUFreq enabled",
+                            "CONFIG_CPU_IDLE" : "CPUIdle enabled",
+                            "CONFIG_SCHED_DEBUG" : "Scheduler Debugging enabled",
+                            #"CONFIG_SCHED_WALT" : "WALT Load Tracking",
+        }
+        fail_list = []
+        message = ""
+        for config in necessary_configs:
+            if not kernel_config.is_enabled(config):
+                fail_list.append(config)
+        if len(fail_list):
+            message = "Configs are missing: "
+            for cfg in fail_list:
+                message = message + "[{} : {}] ".format(
+                    cfg, necessary_configs[cfg])
+        self.assertTrue(len(fail_list) == 0, msg=message)
+
+    def test_SchedTune(self):
+        """
+        Check that SchedTune is present on the target
+        """
+        mount_output = self.target.execute('mount')
+        mount_location = ''
+        SCHEDTUNE_REGEXP = re.compile(r'(?P<location>/\S+).*schedtune\)')
+        DIRENTRY_REGEXP = re.compile(r'd.*[0-9]{2}:[0-9]{2}.(?P<group_name>.+)$')
+        for line in mount_output.splitlines():
+            match = SCHEDTUNE_REGEXP.search(line)
+            if match:
+                mount_location = match.group('location')
+        schedtune = { 'groups': {} }
+        self.assertTrue(len(mount_location) > 0,
+                        "SchedTune CGroup Controller not mounted")
+        # populate groups, start with root group
+        paths = { '' : 'root' }
+        # now find directories
+        dir_list = self.target.execute('ls -l {}'.format(mount_location))
+        for entry in dir_list.splitlines():
+            match = DIRENTRY_REGEXP.search(entry)
+            if match:
+                group_name = match.group('group_name')
+                paths['/{}'.format(group_name)] = group_name
+        # now read the values
+        files = [ 'schedtune.boost' ]
+        for group in paths:
+            schedtune['groups'][paths[group]] = {}
+            for name in files:
+                schedtune['groups'][paths[group]][name] = self.target.read_int(
+                    '{}{}/{}'.format(mount_location, group, name))
+        self.assertTrue(schedtune['groups']['root']['schedtune.boost'] == 0,
+                        "SchedTune Root Group Boost != 0")
+        groups_with_nonzero_boost = 0
+        for groups in schedtune['groups']:
+            if groups == 'root':
+                continue
+            if schedtune['groups'][groups]['schedtune.boost'] != 0:
+                groups_with_nonzero_boost += 1
+        self.assertTrue(groups_with_nonzero_boost != 0,
```

Comment:
Don't think unittest has a warning facility, so I suggest we just keep the test and add some wordiness to the docstring to make it clear that this is not a problem with the kernel or platform but just some configuration that hasn't been done. Maybe eventually there could be separate suites here: one that says "your platform and kernel have the required features" and one that says "you've done the configuration you probably need to do"?